practiceapp/src/main/java/com/practiceapp/practiceapp/controller/views/GetPostsViewController.java

```diff
@@ -27,7 +28,12 @@
     @RequestMapping(path = "/{community_name}",method = RequestMethod.GET)
     public String showPostsByCommunityName(Model model, @PathVariable("community_name") String community_name){  //TEST AFTER POSTS ADDED!
 
-        List<PostEntity> posts = restTemplate.getForObject(base_url + target_url + "/" + community_name, List.class);
+        System.out.println("NEREDE");
+        System.out.println("NEREDE");
+        System.out.println("NEREDE");
```

Comment:
```suggestion
       
```

Reply:
I forgot to delete :D Thanks for noticing! I'm deleting it now.